### PR TITLE
LSFF #1: add population coverage data for folic acid

### PR DESCRIPTION
**Pull request by NathanielBlairStahn**

### CODE REVIEW

File: docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst [hunk 1]
```diff
@@ -104,6 +104,97 @@ Iron Fortification
...
+  * - Nigeria (Kano)
+    - 22.7 (20.0, 25.5)
+    - 83.8 (81.4, 86.2)
+    - 83.9 (81.5, 86.3)
+  * - Nigeria (Lagos)
+    - 5.4 (3.8, 6.9)
+    - 13.8 (11.5, 16.1)
+    - 14.2 (11.8, 16.5)
+
+For Ethiopia, assume
```
Comment (collijk) [inline docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst]: ethiopia
india
nigeria

File: docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst [hunk 2]
```diff
@@ -104,6 +104,97 @@ Iron Fortification
...
+
+.. _GFDx Ethiopia Dashboard: https://fortificationdata.org/country-fortification-dashboard/?alpha3_code=ETH&lang=en
+
+For all the locations other than Ethiopia, use a Beta distribution with mean
+equal to the central estimate, and variance equal to the variance of a normal
+distribution with the same mean and 95% confidence interval.
+
+.. todo::
+
+  Describe how to get the correct Beta distribution.
```
Comment (collijk) [inline docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst]: What I've been doing:
```
def from_statistics(cls, mean, upper_bound, lower_bound, variance=None):
        if variance is None:
            variance = confidence_interval_variance(upper_bound, lower_bound)
        support_width = (upper_bound - lower_bound)
        mean = (mean - lower_bound) / support_width
        variance /= support_width ** 2
        alpha = mean * (mean * (1 - mean) / variance - 1)
        beta = (1 - mean) * (mean * (1 - mean) / variance - 1)
        return  lower_bound + support_width*scipy.stats.beta.rvs(alpha, beta)
```

**Comment by collijk** [comment]
Need guidance on the sampling algorithm.

File: docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst (hunk 2 above)
Comment (NathanielBlairStahn) [reply docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst]: I haven't had a chance to experiment with this yet.  Is getting guidance on this still urgent, or did you come up with something that seems reasonable for now?

The above graphs look reasonable enough to me, but I'd have to check the numbers to see if anything looks off.

Comment (NathanielBlairStahn) [reply docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst]: Another option for India and Nigeria would be to use truncated normal distributions, i.e. just find the normal distribution with the right mean and 95% confidence interval, and truncate the tails outside the interval [0,1] (I don't know off the top of my head how to correctly implement the truncation, though).

**Pull request merged**